Source/WebCore/platform/cocoa/ContentFilterUnblockHandlerCocoa.mm

```diff
@@ -178,14 +179,13 @@
 {
 #if HAVE(WEBCONTENTRESTRICTIONS)
     if (m_evaluatedURL) {
-        if (!m_wcrBrowserEngineClient)
-            m_wcrBrowserEngineClient = adoptNS([PAL::allocWCRBrowserEngineClientInstance() init]);
-        [m_wcrBrowserEngineClient allowURL:*m_evaluatedURL withCompletion:[decisionHandler](BOOL didAllow, NSError *) {
-            callOnMainThread([decisionHandler, didAllow] {
-                RELEASE_LOG(ContentFiltering, "WebFilterEvaluator %s the unblock request.\n", didAllow ? "allowed" : "did not allow");
+        CheckedRef urlFilter = WebCore::ParentalControlsURLFilter::singleton();
+        urlFilter->allowURL(*m_evaluatedURL, [decisionHandler = WTFMove(decisionHandler)](bool didAllow) {
```

Comment:
Why not just use `WebCore::ParentalControlsURLFilter::singleton()` here? Why the need for a CheckedRef variable for a singleton?